src/components/PostPageDetail/elements/Content.js

```diff
@@ -229,7 +229,12 @@ const Content = ({
         ) : null}
         {item && item.post_type === POST_TYPE_LINK && (
           <View
-            style={[styles.newsCard, {marginVertical: handleMarginVertical(sanitizeUrl(message))}]}>
+            style={[
+              styles.newsCard,
+              {
+                marginVertical: handleMarginVertical(sanitizeUrl(message))
+              }
+            ]}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here improve the readability of the code by breaking down the style array into multiple lines. This makes it easier to understand what styles are being applied to the View component.

```diff
- style={[styles.newsCard, {marginVertical: handleMarginVertical(sanitizeUrl(message))}]};
+ style={[
+   styles.newsCard,
+   {
+     marginVertical: handleMarginVertical(sanitizeUrl(message))
+   }
+ ]};
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->